.vscode/settings.json

```diff
@@ -8,5 +8,6 @@
 	],
 	"eslint.validate": ["typescript", "typescriptreact"],
 	"eslint.run": "onSave",
-	"typescript.tsdk": "./node_modules/typescript/lib"
+	"typescript.tsdk": "./node_modules/typescript/lib",
+	"editor.defaultFormatter": "esbenp.prettier-vscode"
```

Comment:
Maybe I should add this to git ignore?

Reply:
No, don't ignore this file. If you want to set personal settings select user settings instead of workspace settings in VSCode.